ClientApp/viking-vault/src/ReponseModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+const modalOpenStyle = {
+    display: "block",
+    opacity: 1,
+    backgroundColor: "rgba(0,0,0,0.5)"
+}
+
+const modalCloseStyle = {
+    display: "none"
+}
+
+class ResponseModal extends React.Component<any, any> {
```

Comment:
`<any, any>` Please use types whenever you can for state and props, I know any is easy to use when debugging but it helps a lot to know later on when you come back to look at the component 2 weeks later what it takes as an input and what data it holds in its state. Also a big advantage is that when you use the component externally you get props autocomplete.